src/app/components/utility/userStateService.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Created by chima on 11/19/15.
+ */
+'use strict'
+
+angular.module('utility')
```

Comment:
I'd suggest this is more appropriately placed in the `auth` component` (or even its own).